include/target.mk

```diff
@@ -176,6 +176,7 @@ ifeq ($(DUMP),1)
     CPU_CFLAGS_24kc = -mips32r2 -mtune=24kc
     CPU_CFLAGS_74kc = -mips32r2 -mtune=74kc
     CPU_CFLAGS_octeon = -march=octeon -mabi=64
```

Comment:
and both are backward compatible to octeon afair. I dont understand what benefit this patch gives us.

Reply:
It mainly adds a few new instructions and as far I can tell by looking at the commit history the support of the original octeon CPUs were dropped in 2013 here
https://github.com/openwrt/openwrt/commit/e7dee4ea8720277cb805919e0fbe54e1ae8c81a2 and was never brought back.

A few details about the difference between Octeon and Octeon+
https://sourceware.org/ml/binutils/2011-11/msg00085.html later comitted as https://sourceware.org/git/gitweb.cgi?p=binutils-gdb.git;a=commit;h=dd6a37e700ab12b5f5e89b747992324e74981872